lib/js/components/Divider/Divider.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+	<div
```

Comment:
Did you try using `<hr>`? It's more semantically correct

Reply:
Even if it's vertical? 🙂

Reply:
you can change the `tagName` based on props ;)

Reply:
To what? There is no `<vr>` 😛  `<hr>` was my first instinct too, but it doesn't fit vertical line. Also:
>The \<hr> HTML element represents a thematic break between paragraph-level elements: for example, a change of scene in a story, or a shift of topic within a section.

I'm not sure if it's even a good match for how we'll be using it.